### DISCUSSION TITLE
NPMPLUS LOCAL-AI not added to installer in latest, beta, or develop

### DISCUSSION THREAD

**Discussion by FlattusBlastus** [nextcloud/all-in-one#3561] [Questions] [answered]
### Steps to reproduce
1.       - AIO_COMMUNITY_CONTAINERS=npmplus vaultwarden local-ai
2. docker compose up -d

### Expected behavior 
Installer should proceed without error
### Actual behavior 
2023-10-18 17:48:55 The community container npmplus was not found!
2023-10-18 17:48:55 The community container local-ai was not found!
2023-10-18 17:48:55 You've set AIO_COMMUNITY_CONTAINERS but at least one container was not found.
2023-10-18 17:48:55 It is set to 'npmplus vaultwarden local-ai'.

### Host OS 
Windows Docker Desktop

#### Nextcloud AIO version 
7.4.1
#### Current channel 
Latest, beta, develop

**Comment by szaimen** [comment, marked as answer]
Hi, both are currently in develop channel. I let you know as soon as both are available in beta channel

**Reply by FlattusBlastus** [reply]
Can confirm I see them in the develop channel now 7.5.0

**Reply by FlattusBlastus** [reply]
Got local ai working. Backup in 7.5.0 doesnt work with local ai on and /models empty. I had to manually download the models. Also, manuals.yaml overwrote itself at some point with a 0 byte file.

![NCA-Im_sorry_letter_gpt4allj](https://github.com/nextcloud/all-in-one/assets/78336592/33b3f167-cfe6-47ae-8eca-94f52a9e1536)

**Reply by szaimen** [reply]
> Backup in 7.5.0 doesnt work with local ai on and /models empty. 

Should be fixed with the next release as models folder are not included then in the backup anymore.

> I had to manually download the models. 

local-ai should take care of downloading as soon as the .yaml file is not empty anymore and you restart the containers.

> Also, manuals.yaml overwrote itself at some point with a 0 byte file.

Really? This should not happen. If it did, I guess it would be a problem of the text app....

**Comment by szaimen** [comment]
This is now released with v7.5.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
